### ISSUE TITLE
How to set encoding for Chinese?

### ISSUE THREAD

**Issue by BananZG** [regexident/DLPDFRenderer#9]
Hi, as mentioned in Title, I was trying to copy the Chinese texts generated with this generator (for checking purpose). Currently all texts copied are show as empty boxes.
I tried some Chinese PDF with proper encoding, they can be copied without any problem.

Is there any way to set the encoding type?

**Comment by regexident** [comment]
Is your content utf8-encoded?

**Comment by BananZGan** [comment]
Hi regexident,
Thanks for replying, my html is utf8-encoded and it is showing correctly in the emulator / ios device.
The Chinese was not showing correctly only when I tried to copy it from the generated pdf.

After some research and testing, I found that the issue is from the version of IOS, nothing to do with the library. Tested on IOS 9.2 and the Chinese is having issue, but on IOS 10.2, everything can be copied correctly.
So I suppose that after IOS 10 and after, the UIPrintFormatter is supporting better Chinese encoding.
